Oracle/PCORNetLoader_ora.sql

```diff
@@ -1713,7 +1713,25 @@ insert into dispensing (
     ,DISPENSE_SUP ---- modifier nval_num
     ,DISPENSE_AMT  -- modifier nval_num
 --    ,RAW_NDC
-)
+)
+/* Below is the Cycle 2 fix for populating the DISPENSING table  */
+select distinct
+  ibf.patient_num patid,
+  null prescribingid,
+  ibf.start_date dispense_date,
+  substr(ibf.concept_cd, 5) ndc,
+  null dispense_sup,
+  null dispense_amt
+from i2b2fact ibf
```

Comment:
Note that the prescribing table pulls facts from the `i2b2medfact` table rather than `i2b2fact`.  `i2b2medfact` is a table containing only medication facts. This was a workaround for of performance problems - see [#523](https://informatics.gpcnetwork.org/trac/Project/ticket/523).  If this new dispensing code runs without major performance issues then it'll give me more confidence that we can remove the `i2b2medfact` workaround.

I'm not suggesting you change anything, just it out.